unittests/rpc/handlers/NoRippleCheckTest.cpp

```diff
@@ -616,14 +616,14 @@ TEST_F(RPCNoRippleCheckTest, NormalPathTransactions)
                     {{
                         "Sequence":{},
                         "Account":"rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-                        "Fee":4,
+                        "Fee":1,
```

Comment:
Note: value changed to 1 because we hardcode it here
```
ON_CALL(*rawBackendPtr, doFetchLedgerObject(ripple::keylet::fees().key, seq, _))
        .WillByDefault(Return(CreateFeeSettingBlob(1, 2, 3, 4, 0)));
```
1 = base
4 = units

@arihantkothari confirmed that actual output is correct by running automation and doing some manual testing.